src/index.html

```diff
@@ -1,14 +1,34 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <h1>Stop watch</h1>
+    <article class="stopwatch">
+      <p class="stopwatch__numbers stopwatch__numbers--0"></p>
+
```

Comment:
```suggestion
```
no need to leave blanks between one-line elements

Reply:
done 
![image](https://user-images.githubusercontent.com/126285904/230851826-0f7b5a81-7603-46b7-917b-5f0abc620e19.png)